R/functions-MSnExp.R

```diff
@@ -519,22 +540,25 @@ combineSpectraMovingWindow <- function(x, halfWindowSize = 1L,
                                        mzFun = base::mean,
                                        intensityFun = base::mean,
                                        mzd = NULL,
-                                       BPPARAM = bpparam()){
+                                       BPPARAM = bpparam(),
```

Comment:
Wouldn't it be better to keep `BPPARAM` as the last parameter? This is its usual position in all functions.

Reply:
Yes, agreed. I'll change.